Integration Tests Failing - Multiple Issue Trackers Enabled

Looks like these are the tests that are failing:

```
FAILED tests/acceptance/standard/test_integrations.py::TestIntegrations::test__if_delete_works - SystemExit: 2
FAILED tests/acceptance/standard/test_integrations.py::TestIntegrations::test__if_push_events_true_works - SystemExit: 2
FAILED tests/acceptance/standard/test_integrations.py::TestIntegrations::test__if_push_events_false_works - SystemExit: 2
FAILED tests/acceptance/standard/test_integrations.py::TestIntegrations::test__if_push_events_change_works - SystemExit: 2
```

Looking at the output, we're getting the same message for all 4 of the tests:

```
"Another issue tracker is already in use. Only one issue tracker service can be active at a time"
```

Looks like this is tied to the fact that both Asana and Redmine are enabled at the same time?

Haven't been able to find any note specifically yet in Gitlab issues/docs that anything in particular has changed. I do see notes [in some older issues](https://gitlab.com/gitlab-org/gitlab/-/issues/328470) though that the constraint on only allowing a single issue tracker to be enabled at a time has been in place for quite some time.

You can see some of this messaging in these PR test results:

https://github.com/gitlabform/gitlabform/actions/runs/10526357975/job/29167036520?pr=814#step:6:802

--- a/tests/acceptance/standard/test_integrations.py
+++ b/tests/acceptance/standard/test_integrations.py
@@ -35,24 +35,20 @@ class TestIntegrations:
                 url: http://foo.bar.com
                 username: foo
                 password: bar
-                active: true
+                active: false
                 commit_events: true
               asana:
                 api_key: foo
                 push_events: true
+                active: true
               slack:
                 webhook: http://foo.bar.com
-                push_events: true
-              redmine:
-                new_issue_url: http://foo.bar.com
-                project_url: http://foo.bar.com
-                issues_url: http://foo.bar.com
                 push_events: true
         """
 
         run_gitlabform(config_integrations, project)
 
-        for integration_name in ["jira", "asana", "slack", "redmine"]:
+        for integration_name in ["jira", "asana", "slack"]:
             integration = project.integrations.get(integration_name)
             assert integration.active is True
 
@@ -83,17 +79,12 @@ class TestIntegrations:
               slack:
                 webhook: http://foo.bar.com
                 push_events: true
-              redmine:
-                new_issue_url: http://foo.bar.com
-                project_url: http://foo.bar.com
-                issues_url: http://foo.bar.com
-                push_events: true
         """
 
         run_gitlabform(config_integration_push_events_true, project)
 
         integrations = []
-        for integration_name in ["asana", "slack", "redmine"]:
+        for integration_name in ["asana", "slack"]:
             integration = project.integrations.get(integration_name)
             integrations.append(integration)
 
@@ -111,17 +102,12 @@ class TestIntegrations:
               slack:
                 webhook: http://foo.bar.com
                 push_events: false # changed
-              redmine:
-                new_issue_url: http://foo.bar.com
-                project_url: http://foo.bar.com
-                issues_url: http://foo.bar.com
-                push_events: false # changed
         """
 
         run_gitlabform(config_integration_push_events_false, project)
 
         integrations = []
-        for integration_name in ["asana", "slack", "redmine"]:
+        for integration_name in ["asana", "slack"]:
             integration = project.integrations.get(integration_name)
             integrations.append(integration)
 
@@ -139,17 +125,12 @@ class TestIntegrations:
               slack:
                 webhook: http://foo.bar.com
                 push_events: true
-              redmine:
-                new_issue_url: http://foo.bar.com
-                project_url: http://foo.bar.com
-                issues_url: http://foo.bar.com
-                push_events: true
         """
 
         run_gitlabform(config_integration_push_events_true, project)
 
         integrations = []
-        for integration_name in ["asana", "slack", "redmine"]:
+        for integration_name in ["asana", "slack"]:
             integration = project.integrations.get(integration_name)
             integrations.append(integration)
 
@@ -166,17 +147,12 @@ class TestIntegrations:
               slack:
                 webhook: http://foo.bar.com
                 push_events: false # changed
-              redmine:
-                new_issue_url: http://foo.bar.com
-                project_url: http://foo.bar.com
-                issues_url: http://foo.bar.com
-                push_events: false # changed
         """
 
         run_gitlabform(config_integration_push_events_false, project)
 
         integrations = []
-        for integration_name in ["asana", "slack", "redmine"]:
+        for integration_name in ["asana", "slack"]:
             integration = project.integrations.get(integration_name)
             integrations.append(integration)
 

